dist/AureliaPlugin.js

```diff
@@ -1,146 +1,146 @@
+            // This plugin adds dependencies traced by html-requires-loader
+            // Note: the config extension point for this one is html-requires-loader.attributes.
+            new HtmlDependenciesPlugin_1.HtmlDependenciesPlugin());
+            if (!opts.noHtmlLoader) {
+                // Ensure that we trace HTML dependencies
+                let module = compiler.options.module;
+                let rules = module.rules || (module.rules = []);
+                // Note that this loader will be in last place, which is important 
+                // because it will process the file first, before any other loader.
+                rules.push({ test: /\.html?$/i, use: "aurelia-webpack-plugin/html-requires-loader" });
+            }
+        }
+        // Common plugins
+        compiler.apply(
+        // Adds some dependencies that are not documented by `PLATFORM.moduleName`
+        new ModuleDependenciesPlugin_1.ModuleDependenciesPlugin({
+            "aurelia-bootstrapper": [
+                opts.aureliaApp,
+                getPAL(compiler.options.target) // PAL for target
+            ],
+            // `aurelia-framework` exposes configuration helpers like `.standardConfiguration()`,
+            // that load plugins, but we can't know if they are actually used or not.
+            // User indicates what he uses at build time in `aureliaConfig` option.
+            // Custom config is performed in use code and can use `.moduleName()` like normal.
+            "aurelia-framework": getConfigModules(opts.aureliaConfig),
+        }), 
+        // This plugin traces dependencies in code that are wrapped in PLATFORM.moduleName() calls
+        new AureliaDependenciesPlugin_1.AureliaDependenciesPlugin(...opts.moduleMethods), 
+        // This plugin preserves module names for dynamic loading by aurelia-loader
+        new PreserveModuleNamePlugin_1.PreserveModuleNamePlugin(), 
+        // This plugin supports preserving specific exports names when dynamically loading modules
+        // with aurelia-loader, while still enabling tree shaking all other exports.
+        new PreserveExportsPlugin_1.PreserveExportsPlugin());
+    }
+}
+exports.AureliaPlugin = AureliaPlugin;
+;
+function getEntry(entry) {
+    // Fix: ideally we would require `entry` to be a string
+    //      but in practice, using webpack-dev-server might shift one (or two --hot) extra entries.
+    if (typeof entry === "object")
+        entry = entry[Object.getOwnPropertyNames(entry)[0]];
+    if (Array.isArray(entry))
+        entry = entry[entry.length - 1];
+    if (typeof entry !== "string")
+        throw new Error("includeAll option only works with a single entry point.");
+    return entry;
+}
+function getPAL(target) {
+    switch (target) {
+        case "web": return "aurelia-pal-browser";
+        case "webworker": return "aurelia-pal-worker";
+        default: return "aurelia-pal-node";

```

Comment:
Mistake, should be `'aurelia-pal-nodejs'`. It would be good to make this configurable. 
When using "electron-renderer" combined with Webpack we most likely want to use `aurelia-pal-browser` too (although `pal-nodejs` should work too).